### PR TITLE
Adds Omni-Phero towers to crash

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Phero towers on crash emit all 3 pheremones at a greater range.
## Why It's Good For The Game
After the addition of phero towers, xenos still struggle a lot in crash due to the typical lack of a Queen.
While this won't necessarily help xenos contest most disks, it will surely aid them a lot when it comes to getting fog camped.
## Changelog
:cl:
add: Pheremone towers on crash emit all 3 types at a wider range.
/:cl:

### CODE REVIEW

**Comment by DeltaFire15** [comment]
Isn't this basically mapwide queen tier phero of all types, given crash silos are distributed over the map and 48 is a HUGE radius? Seems very scuffed for marines to go against.

**Comment by Xander3359** [comment]
> Isn't this basically mapwide queen tier phero of all types, given crash silos are distributed across the map and 48 is a HUGE radius? Seems very scuffed for marines to go against.

Currently the range will rarely go outside of caves.
I could leave the range as it was before (which was 32), but yeah it's basically Queen pheros of all types when near a silo.

**Comment by Joe13413** [comment]
yo uh what the fuck is this
you gotta remember that level 6 warding is +15 soft armor in ALL categories
and level 6 frenzy is about -0.3 to -0.4 slowdown
those are some pretty insane buffs, and i don't really think max level pheros should be basekit

**Comment by Xander3359** [comment]
> those are some pretty insane buffs, and i don't really think max level pheros should be basekit

And I don't think marines should  be able to easily camp fog because xenos have to 4 xenos compared to the 15 marines holding all the 2-tile hallways going in/out of silo.

**Review by Kuro020 — APPROVED** [review]

**Comment by Breadeagle** [comment]
Silo camps can be truly fucked up because the structures inside fog aren't actually immune to explosives. If the tunnel is destroyed xenos can't actually get out without fighting past. In fact this change might miss the fact that structures are destructible but not rebuildable. The pheromone coverage is probably fine but some maps have way better or worse coverage than others. Probably effects small maps a lot more favourably but given that one of them is Orion, which is super marine sided, eh. 

As Closetedskeleton mentions, caste coverage can be really bad due to crash being low pop. They're not likely to even have weeds or walls, much less pheromones.

File: code/modules/xenomorph/xeno_structures.dm [hunk 1]
```diff
@@ -1264,6 +1264,21 @@ TUNNEL
...
 
+/obj/structure/xeno/pherotower/omni
+	desc = "A resin formation that looks like a small pillar. A potent, bizarre smell can be perceived from it."
+	icon_state = "omnitower"
+	aura_radius = 48
+
+/obj/structure/xeno/pherotower/omni/Initialize(mapload, _hivenumber)
+	. = ..() //Don't need to add recov pheros because that's handled by parent
+	current_aura = SSaura.add_emitter(src, AURA_XENO_WARDING, aura_radius, aura_strength, -1, FACTION_XENO, hivenumber)
+	current_aura = SSaura.add_emitter(src, AURA_XENO_FRENZY, aura_radius, aura_strength, -1, FACTION_XENO, hivenumber)
```
[the remark this answers is not in the document]
Comment (psykzz) [reply code/modules/xenomorph/xeno_structures.dm:1275]: +1 for this we need to ensure towers can hold multiple emitters

File: code/modules/xenomorph/xeno_structures.dm [hunk 2]
```diff
@@ -1264,6 +1264,21 @@ TUNNEL
...
+	icon_state = "omnitower"
+	set_light(4, 2, LIGHT_COLOR_WHITE)
+
+/obj/structure/xeno/pherotower/omni/attack_alien(mob/living/carbon/xenomorph/X, damage_amount, damage_type, damage_flag, effects, armor_penetration, isrightclick)
+	return X.balloon_alert(X, "Cannot change") //We're already emitting all the pheros.
```
Comment (psykzz) [inline code/modules/xenomorph/xeno_structures.dm:1280]: The return of balloon_alert shouldn't be returned here.
We should `return FALSE`, to ensure no other action is performed.

**Review by psykzz — CHANGES_REQUESTED** [review]

**Comment by Xander3359** [comment]
After some discussion on discord, this is just a band-aid fix for a fundamentally broken gamemode.
I'm closing this PR because I don't think this actually solves any real issues that the mode has.

**Pull request closed without merging**